src/MockClassifier.Api/Models/Constants.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace MockClassifier.Api.Models
+{
+    public static class Constants
```

Comment:
A lot of this is going to be repeated - it's probably time for us to think about a common library for this kind of thing.

Reply:
💯

Reply:
I've added this issue to track thus. P3 at the moment. https://github.com/buerokratt/Cross-functional-requirements/issues/46